scripts/python/tao_build.py

```diff
@@ -900,10 +900,13 @@ def main():
 
 
 if __name__ == "__main__":
+    exit_code = 0
     try:
         main()
     except Exception as e:
         import traceback
         traceback.print_exc()
+        exit_code = -1
     finally:
         stage_time.report()
+    exit(exit_code)
```

Comment:
what's the difference if we don't use "try" at all?

Reply:
we can print the report at "finally", and also print the traceback at failed, it's easier to debug.

Reply:
Rethrowing the exception in `catch` block could also result to non-zero exit code.

Reply:
> Rethrowing the exception in catch block could also result to non-zero exit code.

@qiuxiafei thanks! Done.